runtime/browser/ui/native_app_window_views.cc

```diff
@@ -158,6 +180,15 @@ void NativeAppWindowViews::Close() {
   window_->Close();
 }
 
+void NativeAppWindowViews::ExitApplication() {
+  auto app_service =
```

Comment:
it should not be like this.. pls add` NativeAppWindowDelegate::OnApplicationExitRequested` (or invent a better name) and propagate it further